GPT/beta-commands/gpt_callables.py

```diff
@@ -0,0 +1,45 @@
+from talon import actions
+from ..lib.HTMLbuilder import Builder
+from typing import Callable
+import inspect
+
+def _generate_tool(func: Callable):
```

Comment:
this appears to be unused?

Reply:
To be used in the future. Right now, tool json specs are manually created, but we can automatically make them. I just don't have time to replace the generation code right now. I think leaving it in is fine for time being.

Reply:
Sounds good. Would be good to have a quick comment to that effect in the code